src/test/java/com/exasol/test/JavaUdfIT.java

```diff
@@ -150,13 +151,39 @@ void testGetTimestampFromSetScript() {
                 + "    " + JAR_INCLUDE_DIRECTIVE + ";\n" //
                 + "    %scriptclass com.exasol.test.testobject.GetTimestampUdf;\n" //
                 + "/\n\n");
+        assertQueryResult("SELECT " + fullyQualifiedScriptName + "(T.V)" + //
+                "FROM VALUES (TO_TIMESTAMP('" + date + "T" + time + "Z', 'YYYY-MM-DDTHH24:MI:SS.FF3Z')) AS T(V)", //
+                table().row(date + " " + time));
+    }
+
+    private static void assertQueryResult(final String sql, final ResultSetStructureMatcher.Builder rowMatcher) {
         try (final Statement statement = connection.createStatement();
-                final ResultSet result = statement
-                        .executeQuery("SELECT " + fullyQualifiedScriptName + "(T.V)" + "FROM VALUES (TO_TIMESTAMP('"
-                                + date + "T" + time + "Z', 'YYYY-MM-DDTHH24:MI:SS.FF3Z')) AS T(V)")) {
-            assertThat(result, table().row(date + " " + time).matches());
+             final ResultSet result = statement.executeQuery(sql)) {
+            assertThat(result, rowMatcher.matches());
         } catch (final SQLException exception) {
-            throw new AssertionError(exception);
+            throw new AssertionError("Unable to assert result of statement: " + sql, exception);
         }
     }
-}
+
+    @Test
+    void testGetSizeFromScalarScript() {
+        final String scriptName = "SIZE_IN_SCALAR_CONTEXT";
+        final String fullyQualifiedScriptName = getFullyQualifiedScriptName(schema, scriptName);
+        executeStatement("CREATE JAVA SCALAR SCRIPT " + fullyQualifiedScriptName + "() RETURNS INTEGER AS\n" //
+                + "    " + JAR_INCLUDE_DIRECTIVE + ";\n" //
+                + "    %scriptclass com.exasol.test.testobject.GetSizeUdf;\n" //
+                + "/\n\n");
```

Comment:
```suggestion
                + "    %scriptclass com.exasol.test.testobject.GetSizeUdf;");
```

Reply:
We need at least one newline or the UDF engine complains.